views/js/controller/ProctorManager/index.js

```diff
@@ -111,7 +111,7 @@ define([
                             } else {
                                 var errorMessage = __('Something went wrong ...');
                                 if(response.proctors){
-                                    errorMessage = __("Unable to revoke all proctors : <ul>");
+                                    errorMessage = __("No authorization to revoke for these proctors : <ul>");
```

Comment:
I know it's not your fault but I disagree with the fact that we use html tags (in this case `ul`) in our translated string. I suggest we use simple formatting likes dashes and newlines to format our translated strings. Just `nl2br` the final error message for nice and clean display.